zerver/webhooks/raygun/view.py

```diff
@@ -0,0 +1,307 @@
+from typing import Any, Dict, Optional, Text
+
+from django.http import HttpRequest, HttpResponse
+
+from zerver.decorator import api_key_only_webhook_view
+from zerver.lib.actions import check_send_stream_message
+from zerver.lib.request import REQ, has_request_variables
+from zerver.lib.response import json_success
+from zerver.models import UserProfile
+
+import time
+
+
+@api_key_only_webhook_view('Raygun')
+@has_request_variables
+def api_raygun_webhook(request: HttpRequest, user_profile: UserProfile,
+                       payload: Dict[str, Any] = REQ(argument_type='body'),
+                       stream: Text = REQ(default='raygun'),
+                       topic: Optional[Text] = REQ(default='test')) \
+        -> HttpResponse:
```

Comment:
Could you format this a bit differently and not use a backslash to break it over two lines? Just bring `) -> HttpResponse:` to be inline with the rest. :)

Reply:
hmm, afraid that the linter won't pass because of [PEP 8](https://www.python.org/dev/peps/pep-0008/#maximum-line-length)? Gonna try it locally first..